chapters/0-introduction.tex

```diff
@@ -156,15 +156,12 @@ \section{How to read this book}
 that contain all the code, data, and meta-information needed
 for others to evaluate and reproduce your work.
 
-\begin{fullwidth}
-\begin{figure}
-	\centering
-	\includegraphics[width=1.25\linewidth]{diagrams/Intro}
-	\caption{}
-	\label{fig:intro}
-\end{figure}
-\end{fullwidth}
-
+Figure \ref{fig:intro} shows a summary of the hands-on data tasks 
+comprising  research data work.
+These are described in chapters 3 to 7.
+Chapters 1 and 2, on the other hand,
```

Comment:
Can we put this before we talk about what each chapter is about? It feels a bit weird that we give a high level over view when we have already talked about the chapters in some details.

Reply:
Sure. Feel free to move it wherever it seems like a better fit, or to remove it altogether.

Reply:
@luizaandrade , I did this in 767ef1359d8c09a62ba0c2463430f5f100b81d10. Is this ok to you?